yarn-project/circuit-types/src/interfaces/pxe.ts

```diff
@@ -53,6 +53,13 @@ import { type SyncStatus, SyncStatusSchema } from './sync-status.js';
  * is exposed to dapps for interacting with the network on behalf of the user.
  */
 export interface PXE {
+  /**
+   * Returns whether an L1 to L2 message is synced by archiver and if it's ready to be included in a block.
+   * @param l1ToL2Message - The L1 to L2 message to check.
+   * @returns Whether the message is synced and ready to be included in a block.
+   */
+  isL1ToL2MessageSynced(l1ToL2Message: Fr): Promise<boolean>;
```

Comment:
I'm not 100% if this is necessary anymore but might be useful to have on the PXE public API.